core/src/main/resources/messages/messages_nl.properties

```diff
@@ -0,0 +1,102 @@
+error.initialization.vector.not.set=Initialization vector is niet gezet
+error.initialization.vector.parameter.null=Parameter initializationVector c=mag niet null zijn
+error.initialization.vector.wrong.size=Lengte van initialization vector ({0}) is fout. Moet zijn {1}
+error.initialization.vector.overwrite=Kan initialization vector niet overschrijven met een andere
+
+error.getting.random.instance=Kan geen randomizer instance krijgen
+error.reading.uuid=Kan het UUID niet lezen
+error.writing.data=Fout bij het schrijven van de data
+
+error.generating.key.pair=Fout bij het genereren van het sleutelpaar
+error.encoding.private.key=Fout bij het coderen van de private key
+error.decoding.private.key=Fout bij het decoderen van de private key
+error.encoding.public.key=Fout bij het coderen van de public key
+error.decoding.public.key=Fout bij het decoderen van de public key
+
+error.buddy.not.found=Vriend met ID ''{0}'' niet gevonden
+error.buddy.user.cannot.be.null=Gebruiker mag niet null zijn
+error.buddy.requesting.user.cannot.be.null=Aanvragende gebruiker mag niet null zijn
+error.buddy.message.entity.cannot.be.null=Message entity mag niet null zijn
+error.buddy.useranonymizedid.cannot.be.null=VPN login ID mag niet null zijn 
+error.buddy.only.twoway.buddies.allowed=Alleen twee-weg vriendschapsrelaties zijn toegestaan
+error.buddy.requesting.user.buddy.is.null=Vriend van de aanvragende gebruiker mag niet null zijn
+error.buddy.accepting.user.is.null=De accepterende gebruiker mag niet null zijn
+
+error.goal.not.found.id=Challenge met ID ''{1}'' voor gebruiker ''{0}'' niet gevonden
+error.goal.cannot.add.second.on.activity.category=Toevoegen tweede challenge in dezelfde categorie ({0}) niet toegestaan.
+error.goal.cannot.remove.mandatory=Verplicht doel met ID ''{1}'' van gebruiker ''{0}'' mag niet verwijderd worden
+
+error.no.device.request.present=Er bestaat geen ''new device request'' voor de gebruiker met het ID ''{0}''
+error.device.request.expired=Het ''new device request'' voor gebruiker met ID ''{0}'' is verlopen
+error.device.request.invalid.secret=Het wachtwoord voor het ''new device request'' is ongeldig. Is het verzoek misschien vervangen door een ander?
+
+error.message.not.found=Bericht met ID ''{0}'' niet gevonden
+error.message.no.dto.manager.registered=Geen DTO manager ''{0}'' geregistreerd
+error.message.action.not.supported=Actie ''{0}'' wordt niet ondersteund
+error.message.anonymized.user.id.must.be.set=Het VPN login ID mag niet leeg zijn
+
+error.loading.activitycategories.from.file=Fout bij het laden van de activiteitencategorien van disk
+error.activitycategory.not.found=Activiteitencategorie met ID ''{0}'' niet gevonden
+error.activitycategory.not.found.by.name=Activiteitencategorie met de naam ''{0}'' niet gevonden
+
+error.ldapuser.action.unknown=Actie ''{0}'' is onbekend
+error.ldapuser.empty.vpn.login.id=Het VPN login ID mag niet leeg zijn
+error.ldapuser.empty.vpn.password=Het VPN wachtwoord mag niet leeg zijn
```

Comment:
Dan is het ook VPN-login-ID. Ik laat dat aan Jan over. Die moet toch alle teksten nog door.